### PR TITLE
Feature 2/electricity branch

### PR DESCRIPTION
**Pull request by gopi-gundapuneni**
Hi Team,

This PR is for adding Feature-2 Electricity files from 6 to 10. Please review and approve the same.

Thanks

### CODE REVIEW

File: Electricity6
Comment (GopiKrishna535) [inline Electricity6:1]: Why this is empty file.

File: Electricity10
Comment (GopiKrishna535) [inline Electricity10:1]: same comment

File: Electricity6
Comment (gopi-gundapuneni) [reply Electricity6:1]: I will add some code later

File: Electricity10
Comment (gopi-gundapuneni) [reply Electricity10:1]: Same comment.

File: Electricity7
Comment (GopiKrishna535) [inline Electricity7:1]: Is this code correct?

**Review by GopiKrishna535 — CHANGES_REQUESTED** [review]

File: Electricity7
Comment (gopi-gundapuneni) [reply Electricity7:1]: yep

**Pull request merged**